test/FlapperUniV2.t.sol

```diff
@@ -420,4 +421,52 @@ contract FlapperUniV2Test is DssTest {
         vm.prank(PAUSE_PROXY); end.cage();
         assertEq(flapper.live(), 0);
     }
+
+    // A shortened version of the sell and deposit flapper that only sells `lot`.
+    // Based on: https://github.com/makerdao/dss-flappers/blob/da7b6b70e7cfe3631f8af695bbe0c79db90e2a20/src/FlapperUniV2.sol
+    function sellLotAndDeposit(PairLike pair, address gem, bool daiFirst, address receiver, uint256 lot) internal {
+
+        // Get Amounts
+        (uint256 _reserveDai, uint256 _reserveGem) = UniswapV2Library.getReserves(UNIV2_FACTORY, DAI, gem);
```

Comment:
I would get rid of the `daiFirst` param and hardcode `gem = MKR` since we are assuming that `_reserveDai == _reserve0` here.

Reply:
Actually we are not assuming that, as `UniswapV2Library.getReserves` sorts the results according to its input tokens.

Reply:
Oh yes ok that's the `UniswapV2Library` version of that function, all good!